include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2089,13 +2089,14 @@ namespace internal
         {
           AssertDimension(
             dof_info.n_vectorization_lanes_filled[dof_access_index][cell],
-            VectorizedArrayType::n_array_elements);
+            VectorizedArrayType::size());
```

Comment:
I think this function (and the next one) would be a place to introduce an abbreviation like `n_simd = VectorizedArrayType::size()`. But that can also be done in another PR.